spec/index.bs

```diff
@@ -27,6 +27,73 @@ Introduction {#intro}
 
 This specification rocks.
 
+Security considerations {#security}
+=====================
+
+<dfn dfn>physical resource</dfn> TEMP
+
+Undefined driver behavior {#security-driver}
+---------------------
+WebGPU implementation translates the workloads issued by the user into API commands specific
+to the target platform. Native APIs specify the valid usage for the commands
+(for example, see [vkCreateDescriptorSetLayout](https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/vkCreateDescriptorSetLayout.html))
+and generally don't guarantee any outcome if the valid usage rules are not followed.
+This is called "undefined behavior", and it can be exploited by an attacker to access memory
+they don't own, or force the driver to execute arbitrary code.
+
+In order to disallow insecure usage, the WebGPU behavior is fully defined. An implementation has to
```

Comment:
> WebGPU behavior is fully defined

This seems a little strong. Maybe "WebGPU's range of allowed behaviors is fully defined"?